extended-tests/build.gradle.kts

```diff
@@ -7,15 +7,14 @@ plugins {
   alias(libs.plugins.kotlin.jvm)
 }
 
-repositories { maven { url = uri("https://oss.sonatype.org/content/repositories/snapshots") } }
-
 dependencies {
   testImplementation(libs.stellar.wallet.sdk)
   testImplementation(libs.okhttp3.mockserver)
   testImplementation(libs.assertj.core)
   testImplementation("org.springframework.boot:spring-boot-starter-web")
   testImplementation(libs.kotlin.serialization.json)
   testImplementation(libs.ktor.client.json)
+  testImplementation(libs.google.gson)
```

Comment:
Is this necessary?

Reply:
It should already be there, but for some reason the code doesn't compile without this library, do you have any idea why?

Reply:
https://github.com/stellar/java-stellar-anchor-sdk/actions/runs/8622916329/job/23635221721
Doesn't compile without it, IMO we should brush out all dependencies in 3.0

Reply:
Lets create a ticket for that, I agree we should do that for 3.0

Reply:
https://stellarorg.atlassian.net/browse/ANCHOR-668